sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_receiver_async.py

```diff
@@ -58,7 +59,9 @@ async def _incoming_transfer(self, frame):
             _LOGGER.debug("<- %r", TransferFrame(payload=b"***", *frame[:-1]), extra=self.network_trace_params)
         self.current_link_credit -= 1
         self.delivery_count += 1
-        self.received_delivery_id = frame[1]  # delivery_id
+        self.received_delivery_id = frame[1] # delivery_id
```

Comment:
Should we maybe only be setting this if `frame[1]` is the first frame in a multi-frame payload?